core-infrastructure/src/db/Core.Database/Scripts/017-CreateBFRTable.sql

```diff
@@ -0,0 +1,13 @@
+IF NOT EXISTS(SELECT *
+              FROM INFORMATION_SCHEMA.TABLES
+              WHERE table_name = 'BFR')
+    BEGIN
+        CREATE TABLE dbo.ComparatorSet

```

Comment:
Table name is incorrect

Reply:
It should be, I'll change that now.

Reply:
Thinking about it BudgetForecastReturn might be a more meaningful name.